data-syncs/building-data-syncs/README.md

```diff
@@ -1,161 +1,138 @@
+The **Source Tab** defines the origin of your data sync. Each data source type, from specific file formats to integrated software systems, requires unique parameters. See the [Supported Data Sources list](../supported-data-sync-sources/) for available options.
 
-3. **The Info tab** defines some basic information about your data sync, such as its name and permissions set _(Image 3)._ This is a role based access system where you can give specific groups read, write, execute, and/or all of the above with admin access. Inputting a name and an Admin Group are mandatory. You also have the optional ability to add in Variables; [please review the documentation here](advanced-settings/variables.md) for more details on Variables.
-
-<figure><img src="../../.gitbook/assets/image (680).png" alt=""><figcaption><p>Image 3: The Info Tab</p></figcaption></figure>
-
-4. **The Source tab** defines important information about the source of your data sync _(Image 4)_. This tab is mandatory. Cinchy supports many different source options including different file types and popular software systems. Each source will have different, and often unique, parameters that must be populated in the Source tab screen. You can review the full list of supported data sources, as well as their unique parameters and features, [here.](../supported-data-sync-sources/)
-
-<figure><img src="../../.gitbook/assets/image (741).png" alt=""><figcaption><p>Image 4: The Source Tab</p></figcaption></figure>
-
-[For event based real-time sync sources](../supported-real-time-sync-stream-sources/) (such as the Cinchy Event Broker/CDC, Kafka Topic, MongoDB Event, Polling Event, Salesforce Platform Event, or the Salesforce Push Topic) you will see an addition configuration tab to configure your Listener. Any configuration populated via the UI here will automatically reconcile back to the Listener Config table in your platform. You are able to set the:
-
-- Topic JSON
-- Connections Attribute(s)
-- Auto Offset Reset
-
-You can find more information about the Listener Config settings in the relevant [Sync Source](../supported-data-sync-sources/) or [Real-Time Sync Stream Source](../supported-real-time-sync-stream-sources/) page(s).
+<figure><img src="../../.gitbook/assets/image (741).png" alt=""><figcaption>Image 4: Setting up the Source Tab</figcaption></figure>
 
 {% hint style="info" %}
-If there is more than one listener associated with your data sync, you will need to configure it via [the Listener Configuration table](../supported-real-time-sync-stream-sources/the-listener-configuration-table.md).
+When working with real-time sync sources, you'll notice an extra tab for Listener configurations. The adjustments you make here directly influence the **Listener Config** table. Navigate through the options and set up as needed. For more info, see the [Listener Config](../supported-real-time-sync-stream-sources/) and [Sync Source](../supported-data-sync-sources/) pages.
```

Comment:
notice an extra _tab_ for Listener configuration

Reply:
Extra tab? what extra tab?

Reply:
There's an **Execution** left sidebar item instead of the **Jobs** item that you would see in Batch syncs.

Reply:
Amended. I still need to see if the annotation block is necessary here.